Automattic/simplenote-android
# Issue/center vector drawables

### Fix
Update vector drawable paths for icons that were not centered.  Most were off by one or two pixels to the top and left.  See the enlarged image with centered squares below as an example of the icons before these changes.

![simplenote_ic_action_remove_24dp](https://user-images.githubusercontent.com/3827611/56879313-0dca6d80-6a0d-11e9-909d-ca2bd55328f5.png)

All the icons were centered in these changes.  See the screenshots below for illustration.

![simplenote_center_create](https://user-images.githubusercontent.com/3827611/56879166-4d448a00-6a0c-11e9-956b-d36c4204bd1c.png)

### Test
1. Open navigation drawer.
2. Notice icons in navigation items are centered.
3. Go to ***All Notes*** list.
4. Notice icon in floating action button is centered.
5. Notice icons in top app bar are centered.
6. Tap note in list.
7. Notice icons in top app bar are centered.

### Review
@daniloercoli, @maxme, and @nbradbury, I requested you as reviewer since you have contributed to Simplenote in the past.  @bummytime, I requested you as a reviewer to keep you informed of Simplenote changes.  Only one developer is required to review these changes, but anyone can perform the review.
